renovate.go

```diff
@@ -0,0 +1,99 @@
+package main
```

Comment:
A bit late to the party. We could move this into some subdirectory, as releases repo structure is getting a bit messy (it has been already before this PR).

Nothing blocking in any case, as we can tidy it at any time.

Reply:
Good input, will do that in another PR 👍🏻